packages/playwright/README.md

```diff
@@ -6,33 +6,9 @@
 
 Install [Node.js](https://docs.npmjs.com/getting-started/installing-node) if you haven't already.
 
-Install Playwright:
+Install Playwright: `npm install playwright`
 
-NPM:
```

Comment:
No need to give yarn and npm examples so made this the consistent style for all the docs.